ExceptionLesson.java

```diff
@@ -0,0 +1,40 @@
+package exceptions;
+
+import java.io.*;
+
+public class ExceptionLesson {
+    public static void main(String[] args) {
+        //#1
+        checkedException();
+        //#3
+        tryWithResource();
+        //#2
+        uncheckedException();
+    }
+
+    public static void checkedException() {
+        File file = new File("not_existing_file.txt");
+        try {
+            FileInputStream stream = new FileInputStream(file);
+        } catch (FileNotFoundException e) {
+            System.out.println(e);
+        }
+    }
+
+    public static void uncheckedException() {
+        int[] arr = {1,2,3,4,5};
+        System.out.println(arr[7]);
+    }
+
+    public static void tryWithResource() {
+        try(TestClass testClass = new TestClass()) {
+            testClass.toString();

```

Comment:
This piece of code returns a string but you don't use it. At least wrap it in System.out.println to see it in console.